helios-client/src/main/java/com/spotify/helios/common/Resolver.java

```diff
@@ -44,11 +48,11 @@ private static String env(final String name, final String defaultValue) {
     return Optional.fromNullable(getenv(name)).or(defaultValue);
   }
 
-  public static Supplier<List<URI>> supplier(final String srvName, final String domain) {
+  Supplier<List<URI>> supplier(final String srvName, final String domain) {
     return supplier(srvName, domain, DEFAULT_RESOLVER);
   }
 
-  static Supplier<List<URI>> supplier(final String srvName, final String domain,
+  Supplier<List<URI>> supplier(final String srvName, final String domain,
```

Comment:
is there a need to keep this as returning `Supplier` after this PR? I see in `PeriodicResolver` you always call `.get()` immediately, and it seems like the only other callers of this method are in the unit test. How about make this just return a `List<URI>`?